src/orders-and-subscriptions/OrdersAndSubscriptionsPage.test.jsx

```diff
@@ -1,33 +1,45 @@
 /* eslint-disable global-require */
 import React from 'react';
-import renderer from 'react-test-renderer';
-import { Provider } from 'react-redux';
-import { IntlProvider } from '@edx/frontend-platform/i18n';
-import configureMockStore from 'redux-mock-store';
+import { render, screen } from '../testing';
 
 import OrdersAndSubscriptionsPage from './OrdersAndSubscriptionsPage';
 
-jest.mock('@edx/frontend-platform/analytics', () => ({
-  sendTrackEvent: jest.fn(),
-}));
-
-const mockStore = configureMockStore();
 const storeMocks = require('../store/__mocks__/mockStore');
 const emptyStoreMocks = require('../store/__mocks__/mockEmptyStore');
 
+const {
+  getByText,
+  getByTestId,
+  queryByText,
+  queryByTestId,
+} = screen;
```

Comment:
I think the component should be mounted when we get/use this method to read the screen values. Does it works for you with the current mounted component down below in `testHeadings` function?

```expect(getByText('My orders and subscriptions')).toBeInTheDocument();```

Reply:
yes it works as expected since I am calling render in the `it()` code block before calling this function